consensus/istanbul/core/handler.go

```diff
@@ -193,7 +199,26 @@ func (c *core) handleCheckedMsg(msg *message, src istanbul.Validator) error {
 	return errInvalidMessage
 }
 
-func (c *core) handleTimeoutMsg() {
+func (c *core) handleTimeoutMsg(nextView *istanbul.View) {
+	// TODO-Klaytn-Istanbul: EN/PN should not handle consensus msgs.
```

Comment:
Is this TODO already done by below condition ?

Reply:
The following logic prevents EN/PN to handle `timeout` message.
The TODO comment means that EN/PN should stop hadlers for all consensus messages including `prepare`, `commit`.